src/components/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,46 @@
+import { Tooltip as MuiTooltip, Typography, TooltipProps } from "@mui/material";
```

Comment:
Nice! I have tooltip in fence project, if they are same or simmilar I can use this component too.